packages/nodes-base/nodes/Xero/Xero.node.ts

```diff
@@ -296,6 +296,7 @@ export class Xero implements INodeType {
 							lineItem.TaxAmount = lineItemValue.taxAmount as string;
 							lineItem.TaxType = lineItemValue.taxType as string;
 							lineItem.UnitAmount = lineItemValue.unitAmount as string;
+							lineItem.Tracking = lineItemValue.tracking as [];
```

Comment:
@abourge what about line 288? I think there is a Tracking object there, could we set this there or should we not do that?

Reply:
@molenchuk Good point, changed it